math/minuit2/inc/Minuit2/MinimumError.h

```diff
@@ -87,9 +91,10 @@ class MinimumError {
    bool HasReachedCallLimit() const { return GetStatus() == MnReachedCallLimit; }
    bool IsAvailable() const { return GetStatus() != MnUnset; }
 
-private:
+public:
```

Comment:
```suggestion
private:
```
Swtiching this `private` to `public` looks like a development artifact to me.